src/log.rs

```diff
@@ -99,7 +99,7 @@ impl EspLogger {
         LevelFilter::from(Newtype(CONFIG_LOG_MAXIMUM_LEVEL))
     }
 
-    pub fn set_target_level(&self, target: impl AsRef<str>, level_filter: LevelFilter) {
+    pub fn set_target_level(target: impl AsRef<str>, level_filter: LevelFilter) {
```

Comment:
Ok, I'll put it back.  Any thoughts on how to access it since LOGGER is static? What about a `pub fn` local to this module that calls `LOGGER.set_target_level()`?